pkg

```diff
@@ -88,20 +89,27 @@ RUN_OPTIONS=(-e RHEL_VERSION)
 
 # Parse command line arguments
 verbose=0
-while getopts :?78bdlpqv opt; do
+debug=0
+quiet=0
+all=0
+build_images=0
+while getopts :?78abdf:lopqv opt; do
 	case $opt in
 		\?)
 			PROJECTS=`$SELF -l | sed "s/^/  - /"`
 			<<-HELP_TEXT cat
 			Usage: $SELF [options] [projects]
-			  -7    Build RPMs targeting CentOS 7 (default)
-			  -8    Build RPMs targeting CentOS 8
-			  -b    Build builder Docker images before building projects
-			  -d    Disable compiler optimizations for debugging.
-			  -l    List available projects.
-			  -p    Pull builder Docker images, do not build them (default)
-			  -q    Quiet mode. Supresses output. (default)
-			  -v    Verbose mode. Lists all build output.
+			  -7           Build RPMs targeting CentOS 7 (default)
+			  -8           Build RPMs targeting CentOS 8
+			  -a           Build all projects, including optional ones.
+			  -b           Build builder Docker images before building projects
+			  -d           Disable compiler optimizations for debugging.
+                          -f <file>    Use <file> as the docker-compose. Default: $COMPOSE_FILE"
+			  -l           List available projects.
+                          -o           Build from the optional list. Same as -f "$COMPOSE_FILE_OPT"
+			  -p           Pull builder Docker images, do not build them (default)
+			  -q           Quiet mode. Supresses output. (default)
+			  -v           Verbose mode. Lists all build output.
```

Comment:
The whitespace should not change for flags that were not added. For flags that were added (`-f` and `-o`), those lines needs to begin with 3 tabs.

Reply:
The whitespace was changed to stay lined up. The addition of a flag with an argument shifted everything to the right. It appears I had some tabs where I thought there were spaces, though, I'll fix that.